src/Lean/Widget/InteractiveGoal.lean

```diff
@@ -136,6 +137,8 @@ def goalToInteractive (mvarId : MVarId) : MetaM InteractiveGoal := do
     let userName? := match mvarDecl.userName with
       | Name.anonymous => none
       | name           => some <| toString name.eraseMacroScopes
-    return { hyps, type := goalFmt, userName? }
+    let mut pref := "⊢ "
+    if isLHSGoal? mvarDecl.type |>.isSome then pref := "| "
```

Comment:
```suggestion
    -- use special prefix for `conv` goals
    if isLHSGoal? mvarDecl.type |>.isSome then pref := "| "
```
Let's add a quick comment here since it may not be immediately clear what `isLHSGoal?` is testing.